recipes/compas_nurbs/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "compas_nurbs" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3c9e9d3aba033990d4024cafc4db10148455764652dc2cde935bc8208e76c99e
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - python >=3
+    - compas >=1.0,<2.0
+
+test:
+  imports:
+    - compas_nurbs
+
+about:
+  home: https://github.com/gramaziokohler/compas_nurbs
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: NURBS for COMPAS
+  description: |
+    compas_nurbs is a package inspired by the NURBS-Python,
+    however uses a NumPy-based backend for better performance.
+    Curve, and Surface are non-uniform non-rational B-Spline geometries (NUBS),
+    RationalCurve, and RationalSurface are non-uniform rational B-Spline Geometries (NURBS).
+    They all built upon the class BSpline. Coordinates have to be in 3D space (x, y, z).
+  doc_url: https://gramaziokohler.github.io/compas_nurbs
+  dev_url: https://github.com/gramaziokohler/compas_nurbs
+
+extra:
+  recipe-maintainers:
+    - romanarust
+    - beverlylytle
```

Comment:
@romanarust and @beverlylytle are you OK being added as maintainers here?

Reply:
Yes, I am.

Reply:
me too